src/integrator_hermes.c

```diff
@@ -60,6 +60,7 @@ void reb_integrator_hermes_part1(struct reb_simulation* r){
     mini->collision_resolve = r->collision_resolve;
     mini->collision_resolve_keep_sorted = r->collision_resolve_keep_sorted;
     mini->track_energy_offset = r->track_energy_offset;
+    mini->force_is_velocity_dependent = global->force_is_velocity_dependent;
```

Comment:
This is where you could also copy they pointer to the post-timestep-modifications:  `mini->post_timestep_modifications = global->post_timestep_modifications;`